draft-ietf-cbor-7049bis.md

```diff
@@ -1086,7 +1092,10 @@ subsequent values are zero or more floating-point numbers" or "the
 item is a map that has byte strings for keys and contains at least one
 pair whose key is 0xab01".
 
-This specification puts no restrictions on CBOR-based protocols. An
+CBOR-based protocols MUST specify how their decoders handle
+invalid and other unexpected data.  CBOR-based protocols
+MAY specify that they treat arbitrary valid data as unexpected.
+Encoders for CBOR-based protocols MUST produce only valid items. An
```

Comment:
valid?
That may be out of the reach of the encoder.

Reply:
I think for a *protocol* encoder, it's in reach. It wouldn't be in reach for a generic CBOR encoder, but that's not what I'm talking about here.